cfme/rest/__init__.py

```diff
@@ -168,7 +168,7 @@ def rates(request, rest_api):
     chargeback = rest_api.collections.chargebacks.get(rate_type='Compute')
     data = [{
         'description': 'test_rate_{}_{}'.format(_index, fauxfactory.gen_alphanumeric()),
-        'rate': 1,
```

Comment:
I guess it corresponded to settings in UI where you can specify Fixed rate or Variable rate.
As described in https://github.com/ManageIQ/manageiq/issues/10501 current API `/rates` does not support tiered rates at the moment and therefore "is hardly usable".